docs/marketplace-and-submit-job-v2/marketplace-and-submit-job-v2.md

```diff
@@ -79,40 +79,280 @@ And you can share the script, data, docker images or whole job configuration on
 
 
 
-### Introduction to yaml file
+### Introduction to the yaml file for job configuration
 
-In submit job v2,  we use **yaml** to describe the job configuration.
+In submit job v2,  we use **yaml** to describe the job configuration. The detailed format is shown as below :
 
-Below is an example for tensorflow image classification :
+> `job`
+
+```yaml
+protocol_version: String
+name: String
+type: job
+version: String
+contributor: String
+description: String
+retryCount: Integer
+virtualCluster: String
+gpuType: String
+
+parameters:
+  userParam1: valType1
+  userParam2: valType2
+  ...
+
+tasks:
+  - task
+  - task
+  ...
+
+prerequisites: 
+  - dockerimage
+  - script
+  - data
+  ...
+```
+
+> `data`
+
+```yaml
+protocol_version: String
+name: String
+type: data
+version: String
+contributor: String
+description: String
+uri: String
+```
+
+> `script`
+
+```yaml
+protocol_version: String
+name: String
+type: script
+version: String
+contributor: String
+description: String
+uri: String
+```
+
+> `dockerimage`
+
+```yaml
+protocol_version: String
+name: String
+type: dockerimage
+version: String
+contributor: String
+description: String
+uri: String
+```
+
+> `task`
```

Comment:
>task [](start = 3, length = 4)

task is a concept need more explanation. maybe you can add some introduction before the detailed format. 
e.g., a job may consists of multiple types of tasks, denoted by task role. Each task role container one or more containers, the instantiation of task role.